settings.gradle

```diff
@@ -57,9 +57,10 @@ List projects = [
   'plugins:store-smb',
   'qa:backwards-5.0',
   'qa:evil-tests',
+  'qa:multi-cluster-search',
   'qa:no-bootstrap-tests',
   'qa:rolling-upgrade',
-  'qa:multi-cluster-search',
+  'qa:smoke-test-autocreateindex-limited',
```

Comment:
Of course I have a complaint about the naming. 😇 

I think that we should stop calling these "smoke tests" as they are not smoke tests, and I would like to see some extra hyphens in `autocreateindex`.